### ISSUE TITLE
Error while loading terasology

### ISSUE THREAD

**Issue by adeleglise** [MovingBlocks/Terasology#2128]
http://pastebin.com/DrZ5XfZU

Running on linux mint 17.2 x86_64 , Alienware M17x-R4, AMD [AMD/ATI] Wimbledon XT [Radeon HD 7970M] , Java(TM) SE Runtime Environment (build 1.8.0_65-b17)

**Comment by MarcinSc** [comment]
Seems another case of enabling all modules. Could we please at least consider adding a huge warning (in red and blinking) that discourages manually adding modules?

**Comment by Cervator** [comment]
Yep this is from having all modules enabled, specifically Rails which is not in the stable lineup but available for download from the meta server via in-game module browser.

@MarcinSc Good advice and fully agreed. I added #2128 just now for the easy additions. There are two older issues that relate to the more advanced options. #1911 for custom gameplay templates and #1891 for filtering obsolete modules from the meta server.

I think we should change or remove the Modules button later, closer to beta (v2.0.0), when we've fleshed out module combinations better. Optional/supporting dependencies, exclusive modules (only work as part of a template), module categories, etc. That could help filter the list, if we still expose it directly.
